modules/utilities.py

```diff
@@ -592,11 +593,13 @@ def get_problems_table(all_problems,
                                     problem["link"],
                                     link_class,
                                     link_title,
-                                    problem["id"]),
-                     _class="problem-search-name-column"))
+                                    problem["id"],
+                                    page_prefix=page_prefix),
+                     _class=generate_page_specific_class(page_prefix, "name-column")))
         tr.append(TD(A(I(_class="fa fa-link"),
                        _href=problem["link"],
-                       _class="tag-problem-link",
+                       _class=generate_page_specific_class(page_prefix, "tag-problem-link") + " tag-problem-link",
```

Comment:
```suggestion
                       _class=generate_page_specific_class(page_prefix, "tag-problem-link"),
```

Reply:
We discussed that let the original tag be there as well so that we can continue tracking it. Otherwise, there will be a sudden dip in the graph.